integrated_channels/integrated_channel/exporters/learner_data.py

```diff
@@ -317,43 +326,52 @@ def export(self, **kwargs):
                     self._collect_certificate_data(enterprise_enrollment, channel_name)
                 LOGGER.info(generate_formatted_log(
                     channel_name, enterprise_customer_uuid, lms_user_id, course_id,
-                    '_collect_certificate_data finished with CompletedDate: {completed_date},'
-                    ' Grade: {grade}, IsPassing: {is_passing},'
-                    .format(
-                        completed_date=completed_date_from_api,
-                        grade=grade_from_api,
-                        is_passing=is_passing_from_api
-                    )))
+                    f'_collect_certificate_data finished with CompletedDate: {completed_date_from_api},'
+                    f' Grade: {grade_from_api}, IsPassing: {is_passing_from_api},'
+                ))
             # For self-paced courses, check the Grades API
             else:
-                completed_date_from_api, grade_from_api, is_passing_from_api, grade_percent = \
-                    self._collect_grades_data(enterprise_enrollment, course_details, is_audit_enrollment, channel_name)
+                completed_date_from_api, grade_from_api, is_passing_from_api, _ = \
+                    self._collect_grades_data(enterprise_enrollment, course_details, channel_name)
                 LOGGER.info(generate_formatted_log(
                     channel_name, enterprise_customer_uuid, lms_user_id, course_id,
-                    '_collect_grades_data finished with CompletedDate: {completed_date},'
-                    ' Grade: {grade}, IsPassing: {is_passing},'
-                    .format(
-                        completed_date=completed_date_from_api,
-                        grade=grade_from_api,
-                        is_passing=is_passing_from_api
-                    )))
+                    f'_collect_grades_data finished with: CourseMode: {enterprise_enrollment.mode}, '
+                    f' CompletedDate: {completed_date_from_api},'
+                    f' Grade: {grade_from_api},'
+                    f' IsPassing: {is_passing_from_api},'
+                    f' Audit Mode?: {is_audit_enrollment}'
+                ))
 
             # Apply the Source of Truth for Grades
-            grade = grade_from_api
-            completed_date = completed_date_from_api
-            is_passing = is_passing_from_api
             records = self.get_learner_data_records(
                 enterprise_enrollment=enterprise_enrollment,
-                completed_date=completed_date,
-                grade=grade,
-                is_passing=is_passing,
-                grade_percent=grade_percent
+                completed_date=completed_date_from_api,
+                grade=grade_from_api,
+                course_completed=is_course_completed(
+                    enterprise_enrollment,
+                    completed_date,
+                    is_passing_from_api,
+                    incomplete_count,
```

Comment:
`incomplete_count` will only be present if this is an audit enrollment, right?

Reply:
yup, the function that uses this value validate for None, so if we pass None for audit case it will fail in case we pass None for audit